### DISCUSSION TITLE
Problem in Checkbox and Pagination in Data Tables

### DISCUSSION THREAD

**Discussion by ghadaalfaisal** [livewire/livewire#3931] [1. Help]
Greeting all,
I'm currently struggling with checkbox and pagination in data tables, 
When I go to the next page and select checkbox the page redirects me to the first page

**here is my pagination blade, I replace all href=# to avoid reloading the page**
https://laptrinhx.com/laravel-8-search-with-pagination-using-livewire-1024914969/ 

**here is the data table with a checkbox input, how can I fix this issue without the need to use ?page=num because it gives me an error**

`    <table class="table table-bordered " id="" width="100%" cellspacing="0">
                <thead>
                    <tr>
                        <th><input type="checkbox" wire:model=""></th>  
                        <th>ID</th>  
                        <th>details</th>
                    </tr>
                </thead>

                <tbody>
                    @foreach ($IPs as $IP)
                    <tr wire:key="row-{{ $IP->id }}">
                        <td> <input type="checkbox" value="{{ $IP->id }}" wire:model="selectedIPs"></td>
                        
                        <td>{{ $IP['id'] }}</td>
                        <td>{{ $IP['details'] }}</td>
                    </tr>
                    @endforeach
 </tbody>
            </table>`

**Comment by jonneroelofs** [comment]
You probably want to remove your setPage function. I think the idea of the WithPagination trait is that it handles setting the correct page. My guess is that you are accidentally messing that up now.

**Reply by ghadaalfaisal** [reply]
Thank you so much for your help

**Comment by denizozturk** [comment]
I have no idea why you want to do this, but if you insist on doing it, you can include the line or input as a new livewire component. Alternatively you can use a js plugin instead of livewire pagination.

**Reply by ghadaalfaisal** [reply]
Yes, I used js plugin and it works. When I used livewire pagination I got this error 

The GET method is not supported for this route. Supported methods: POST in Laravel livewire

and I didn't know how to fix it in another way than href="#"

**Comment by joshhanley** [comment]
@ghadaalfaisal that article is pretty dated not, as Livewire's pagination templates use buttons, not `<a>` tags, for that reason. But there are ways to get the `<a>` tags working if needed

If you can share your pagination template you are currently using, maybe we can see if it can be fixed up?

Hope this helps!
